lib/generators/decidim/templates/initializer.rb

```diff
@@ -3,4 +3,5 @@
   config.application_name = "My Application Name"
   config.mailer_sender    = "change-me@domain.org"
   config.authorization_handlers = [ExampleAuthorizationHandler]
+  # config.available_locales = %{en ca es}
```

Comment:
Maybe we can add a comment like Rails does in their config file then? Something like

```ruby
# Uncomment this lines to set your preferred locales
# config.available_locales = %{en ca es}
```

What do you think about this?